### DISCUSSION TITLE
Why not use a kubernetes job for the db init?

### DISCUSSION THREAD

**Discussion by dim-ops** [apache/airflow#29976] [Q&A]
Hello 👋 

I don't understand the choice to setup `useHelmHooks: false` for the first deployment and setup `useHelmHooks: true` after. Why not use a kubernetes job with  `if eq .Release.Revision "1"` ? Like this helm chart is idempotent.

I can do the PR if necessary.

Thank your for your futur reply.

**Comment by potiuk** [comment]
If you can propose a better approach for hooks - feel free to open PR. I think indeed the current Hook approach has limitations, Discussing it in a PR with tests and explanation is better than theorising here.
